### PR TITLE
[WIP] Bert horovod for Summit

### PR DESCRIPTION
**Pull request by yngtodd**
Working on making Bert easier to use on Summit.

### CODE REVIEW

**Comment by gounley** [comment]
One annoying comment: that conda is going to need to live in MED106, not MED107.

**Comment by yngtodd** [comment]
> One annoying comment: that conda is going to need to live in MED106, not MED107.

One of the main questions that I have is how we manage the [Bert model weights](https://github.com/yngtodd/Benchmarks/blob/develop/Pilot3/P3B6/launch_single.lsf#L17).

**Comment by jmohdyusof** [comment]
If you are actively developing, is there a reason you can't just commit to develop directly?

**Comment by gounley** [comment]
Dang, I didn't think about the model weights. Could we those on the same FTP server as the synthetic data?

**Comment by yngtodd** [comment]
> If you are actively developing, is there a reason you can't just commit to develop directly?

haha I actually tried that first, but found that I was getting permission denied when pushing to `Benchmarks/develop`.

File: Pilot3/P3B6/horovod.lsf [hunk 1]
```diff
@@ -12,9 +12,11 @@ module load gcc
 module load open-ce
 module unload darshan
 
-conda activate /gpfs/alpine/proj-shared/med107/yngtodd/envs/opence
+MED106="/gpfs/alpine/proj-shared/med106"
+ENV_PATH="$MED106/envs/opence"
+WEIGHTS_PATH="$MED106/weights/ncbi_bert_base_pubmed_mimic_uncased"
+CMD="python single_node.py --pretrained_weights_path $WEIGHTS_PATH"
```
Comment (yngtodd) [inline Pilot3/P3B6/horovod.lsf:19]: Everyone on `med106` should be able to kick this off now.

**Comment by yngtodd** [comment]
> Dang, I didn't think about the model weights. Could we put those on the same FTP server as the synthetic data?

Are there some docs that you can point me to for this?

**Comment by jmohdyusof** [comment]
There should be no reason you can't push to develop, it is only master that is protected. John can you push to develop?

**Comment by yngtodd** [comment]
When I talked with John about the permissions, it seemed like that might have been a recent change per one of our calls a couple weeks back. I also tested creating a new branch, to see if I just didn't have permission to push to `develop`, but I couldn't push that new branch either.

**Comment by jmohdyusof** [comment]
Tom did some stuff with permissions to make sure people didn't push to master, so it might have changed develop by mistake. Can you ask Tom on slack or email to give you permissions to create branches and push to develop

**Comment by gounley** [comment]
@yngtodd The data is stored at https://ftp.mcs.anl.gov/pub/candle/public/benchmarks/ but I don't know who has access. Maybe @hyoo ?

**Comment by yngtodd** [comment]
> Tom did some stuff with permissions to make sure people didn't push to master, so it might have changed develop by mistake. Can you ask Tom on slack or email to give you permissions to create branches and push to develop

I can ask him about that. Of course, if you would rather me work directly on `develop`, we will need to pull in these changes, and then I can start from there.

**Comment by jmohdyusof** [comment]
Yeah, if you get permission we can then merge this and you can go from there. If for some reason he doesn't want to give you permission (?) then keep committing here and I will do a merge once it is complete.

**Comment by yngtodd** [comment]
That sounds good to me!

**Comment by jmohdyusof** [comment]
Merge for now.

**Pull request merged**